specification/redis/resource-manager/readme.md

```diff
@@ -76,4 +76,40 @@ csharp:
   namespace: Microsoft.Azure.Management.Redis
   output-folder: $(csharp-sdks-folder)/RedisCache/Management.Redis/Generated
   clear-output-folder: true
-```
\ No newline at end of file
+```
+
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: cache
+  clear-output-folder: true
+```
+
+### Tag: package-2017-02 and go
+
+These settings apply only when `--tag=package-2017-02 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-02' && $(go)
+output-folder: $(go-sdk-folder)/services/redis/mgmt/2017-02-01/cache

```

Comment:
>redis/mgmt/2017-02-01/cache [](start = 41, length = 27)

Why not `redis/mgmt/2017-02-01/redis` ?

Reply:
Because I'm trying to match the Swagger declarations.